LambaAnalyzer/src/LambdaAnalyzer.cc

```diff
@@ -1429,6 +1436,19 @@ void LambdaAnalyzer::beginJob(){
     //tree1->Branch("KpiTrkSpt",&KpiTrkSpt);
     tree1->Branch("LambdaPt",&LambdaPtKpi);
     tree1->Branch("MCLambdaDeltaR",&MCLambdaDeltaR);
+
+    // Global position vars of rec Lambda added to tree
```

Comment:
This info is actually already being saved. See https://github.com/nhaubrich/TrackingDstar/blob/24150c7fb22efd3970b653ce515a7c3d64257ed7/LambaAnalyzer/src/LambdaAnalyzer.cc#L1355-L1363